src/jsdocUtils.js

```diff
@@ -186,20 +186,26 @@ const potentiallyEmptyNamepathTags = [
   'event',
   'callback',
   'class', 'constructor',
+  'extends', 'augments',
   'constant', 'const',
   'function', 'func', 'method',
   'interface',
   'member', 'var',
   'mixin',
   'namespace',
-  'listens', 'fires', 'emits'
+  'listens', 'fires', 'emits',
+  'see',
```

Comment:
As mentioned below `@see` has special handling.

Reply:
Now 'see' is in `tagsWithOptionalNamepath` list as it may have a namepath or may not have it. Its handling is done within `valid-types` rule.